### DISCUSSION TITLE
Github pages show "CNAME is already taken" although it is not.

### DISCUSSION THREAD

**Discussion by thesagniksaha** [community/community#22705] [Other Feature Feedback, Questions, & Ideas]
<p>I want to host the repository <a href="https://github.com/organisation/organisation.github.io" rel="nofollow noopener">https://github.com/organisation/organisation.github.io</a> on github pages and use the custom subdomain name.domain.ac.in. I added a CNAME record for <a href="http://organisation.github.io" rel="nofollow noopener">organisation.github.io</a> in my domain’s DNS settings, but when I tried to add a CNAME record for the subdomain in the respository, it shows: The CNAME name.domain.ac.in is already taken. There are no other repositories using this CNAME currently, so I can’t figure out the problem. Please help.</p>

**Comment by logankilpatrick** [comment]
<p>Hey <a class="mention" href="/thesagniksaha">@thesagniksaha</a>, it looks like this got resolved, is there any suggestions you can write up for folks with the same issue you had? Thanks! </p>

**Comment by thesagniksaha** [comment, marked as answer]
<p>The problem that happened here was a case of subdomain takeover, and is a serious issue. If you create a subdomain CNAME record in your domain’s DNS settings that points to a github username and don’t update the custom domain address in your repository’s  Settings &gt; GitHub Pages within a stipulated amount of time, your subdomain could be taken over by someone else, and if you try to add that subdomain to a repository later, it will show that “The CNAME <a href="http://subdomain.domain.com" rel="nofollow noopener">subdomain.domain.com</a> is already taken”.</p>
<p>Patrik has written a lot about subdomain takeovers in his blog here: <a href="https://0xpatrik.com/" rel="nofollow noopener">https://0xpatrik.com/</a></p>
<p>Hope this was useful.</p>

**Comment by jmmoore** [comment]
<p>Were you able to resolve the takeover? How did you determine that it had been taken over in the first place?</p>

**Comment by lczzlczz** [comment]
<p>I have the same issue yet to be resolved.</p>

**Comment by archived-aft-bogus-german-visa-rejectio** [comment]
<p>Got the same error as well when I tried to use my custom domain “lo.fish”. Maybe a cache problem?</p>
<p>Looked up on Stack Overflow and tried everything but to no avail.</p>
<p>here is the repo: <a href="https://github.com/CAST3LLA/lo.fish/" rel="nofollow noopener">https://github.com/CAST3LLA/lo.fish/</a></p>
